templates/omw.html

```diff
@@ -0,0 +1,20 @@
+{% set omw = site.get('/omw') %}
```

Comment:
This line is the magic sauce to get the single-page style to work with Lektor smoothly.  Otherwise, _all_ content would have to live in the same model and the same content file.